### DISCUSSION TITLE
QA: How to monitor Host's network?

### DISCUSSION THREAD

**Discussion by gerroon** [librenms/docker#371] [Q&A]
Hi

I would like to monitor the net traffic of the docker host(one docker-compose is running it), I have it up and running but I am not sure about my next step. Basically I want to be able to see the network traffic that this host laptop engages in.

thanks

**Comment by murrant** [comment]
Add the host to LibreNMS.
https://docs.librenms.org/Support/SNMP-Configuration-Examples/#linux-snmpd
